core-services/prow/02_config/_plugins.yaml

```diff
@@ -3728,6 +3735,27 @@ plugins:
   - wip
   - yuks
   - approve
+  jianzhangbjz/learn-operator:
+  - assign
+  - blunderbuss
+  - cat
+  - dog
+  - heart
+  - golint
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
+  - shrug
+  - sigmention
+  - skip
+  - trigger
+  - verify-owners
+  - wip
+  - yuks
+  - approve
```

Comment:
You have enabled merge-related plugins such as `lgtm` or `approve` but not automated merges, is that intentional?

Also, how will you make sure your repository has all necessary labels?

Reply:
> but not automated merges

I'm sorry that I'm confused. I configure these plugins based on the https://prow.k8s.io/plugins. Which plugins I missed? Thanks! I thought the Prow will merge the PR automatically when get the `/approve` and `/lgtm` labels.

> Also, how will you make sure your repository has all necessary labels?

I configure my repo by following this: https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#repository-configuration. And, I just add those labels based on my experiences. Anything else I need to add or remove? Thanks!

Reply:
Merge configuration is documented in that doc, specifically in  https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#enabling-automatic-merges

Reply:
Thanks! My misunderstanding, add it.